tests/Plan/ConfigPlanRepositoryTest.php

```diff
@@ -83,7 +83,7 @@ public function populatesPlanProperlyWithoutDefaultsSet()
 
         $this->assertNull($plan->firstPaymentDescription());
         $this->assertNull($plan->firstPaymentAmount());
-        $this->assertNull($plan->firstPaymentMethod());
+        $this->assertNull($plan->paymentMethod());
```

Comment:
Why the rename?

Reply:
If we have methods defined in the plan config we use it, if not we use first payment methods. I keep firstPaymentMethod to access payments method(s) from config of first payment method. `paymentMethod()` !== firstPaymentMethod()`